runtime/src/iree/schemas/hip_executable_def.fbs

```diff
@@ -6,11 +6,11 @@
 
 include "iree/schemas/executable_debug_info.fbs";
 
-namespace iree.hal.rocm;
+namespace iree.hal.hip;
 
-// 'ROCM Executable'.
-file_identifier "ROCM";
-file_extension "rocm";
+// 'HIP Executable'.
+file_identifier "HIPE";
+file_extension "hipe";
```

Comment:
Could this use just "HIP" instead of "HIPE"? Do the other executables have an "E" suffix?

Reply:
file_identifier has to be 4 characters long (it becomes a uint32_t in the file format)

Reply:
genius idea: `HIP1` so we have a good way to version